queries/agenda.js

```diff
@@ -178,11 +189,19 @@ const fetchDecisionsFromAgenda = async (agendaId, currentUser, extensions) => {
           mu:uuid ${sparqlEscapeString(agendaId)} ;
           dct:hasPart ?agendaitem .
       ?agendaitem a besluit:Agendapunt ;
-          ^dct:subject/besluitvorming:heeftBeslissing/^besluitvorming:beschrijft ?document .
+          ^dct:subject/besluitvorming:heeftBeslissing/^besluitvorming:beschrijft ?originalDocument .
+      ?originalDocument prov:value / ^prov:hadPrimarySource? ?originalFile .
+
+      OPTIONAL {
+          ?originalDocument sign:getekendStukKopie ?flattenedDocument .
+          ?flattenedDocument prov:value ?flattenedFile . 
+      }
```

Comment:
this optional needs both of these, we coud have a situation where the flattendDoc is in the graph, but the file isn't.